packages/browser/src/sessionid.ts

```diff
@@ -36,6 +37,11 @@ export class SessionIdManager {
     // we track activity so we can end the session proactively when it has passed the idle timeout
     private _enforceIdleTimeout: ReturnType<typeof setTimeout> | undefined
 
+    private _eventEmitter: SimpleEventEmitter = new SimpleEventEmitter()
+    public on(event: 'forcedIdleReset', handler: () => void): void {
+        this._eventEmitter.on(event, handler)
+    }
```

Comment:
adding an event emitter now, it's hard to move old events over to it as lots of the old events assume they might be called immediately on registering a handler which the event emitter doesn't do, it only fires on the next trigger